### PR TITLE
feat: Add download ticket option for each attendee

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #

#### Short description of what this resolves:
feat: Add download ticket option for each attendee

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/forms/orders/attendee-list.hbs [hunk 1]
```diff
@@ -14,6 +14,11 @@
             <i class="user icon"></i>
             <label>{{t 'Ticket Holder '}}{{t ' -for- '}}{{holder.ticket.name}}</label>
           </div>
+          <button class="ui labeled icon {{if this.device.isMobile 'fluid'}} button"
+          {{action this.downloadTicketForAttendee holder.event.name holder.order.id holder.id }}>
```
Comment (codedsun) [inline app/templates/components/forms/orders/attendee-list.hbs:18]: @maze-runnar  - here the holder.event.name when downloading ticket is undefined, but when i print in console, it gives me the value of the event. Please help

Comment (codedsun) [reply app/templates/components/forms/orders/attendee-list.hbs:18]: Also how do I align to end this button?
image

**Review by daretobedifferent18 — COMMENTED** [review]
instead of using it ```holder.event.name``` you can use ```this.event.name``` and for ```holder.order.id``` use ```this.data.id```

and also you can define download action in new.js where other download actions are there so that everything is together

File: app/templates/components/forms/orders/attendee-list.hbs (hunk 1 above)
Comment (maze-runnar) [reply app/templates/components/forms/orders/attendee-list.hbs:18]: > Also how do I align to end this button?
> image

`<button class="ui right floated button">Right Floated</button>`

Comment (maze-runnar) [reply app/templates/components/forms/orders/attendee-list.hbs:18]: > @maze-runnar - here the holder.event.name when downloading ticket is undefined, but when i print in console, it gives me the value of the event. Please help

`@event.name` for event name.

Comment (codedsun) [reply app/templates/components/forms/orders/attendee-list.hbs]: Can you also xplain @maze-runnar plz

**Comment by codedsun** [comment]
@iamareebjamal - review

**Comment by iamareebjamal** [comment]
Ticket Holder download not working. Server returns 404 hence corrupted PDF

File: app/templates/components/forms/orders/attendee-list.hbs (hunk 1 above)
Comment (maze-runnar) [reply app/templates/components/forms/orders/attendee-list.hbs]: in order/view.hbs

```
<Forms::Orders::AttendeeList
        @save="save"
        @data={{this.model.order}}
        @fields={{this.model.form}}
        @event={{this.model.event}} />
```

**Pull request merged**